eventmesh-connectors/eventmesh-connector-spring/build.gradle

```diff
@@ -15,22 +15,16 @@
  * limitations under the License.
  */
 
-configurations {
-    implementation.exclude group: 'ch.qos.logback', module: 'logback-classic'
-    implementation.exclude group: 'log4j', module: 'log4j'
-    testImplementation.exclude group: 'org.apache.logging.log4j', module: 'log4j-to-slf4j'
-}
-
 dependencies {
     api project(":eventmesh-openconnect:eventmesh-openconnect-java")
     implementation project(":eventmesh-common")
     implementation project(":eventmesh-sdks:eventmesh-sdk-java")
-    implementation "org.springframework.boot:spring-boot-starter:$spring_boot_version"
-    implementation "org.springframework.boot:spring-boot-starter-validation:$spring_boot_version"
-    implementation "org.springframework:spring-messaging:$spring_version"
+    implementation "org.springframework.boot:spring-boot-starter-validation"
+    implementation "org.springframework:spring-messaging"
+
```

Comment:
Why don't we exclude `spring-boot-starter-logging` instead of removing `spring-boot-starter` here?

@yanrongzhen Please have a check

Reply:
The removal of `spring-boot-starter` was not intentional, I'll add it again.